File: workspace/pset6/mario/mario.py
from cs50 import get_int

height = get_int("Height: ")

while(height < 0 or height >= 24):
    height = get_int("Height")

leftBlanks = height - 1
rightBlanks = (height * 2) - (height - 1)

i = 0
j = 0

for i in range(height):

    for j in range((height * 2) + 2):

        if(j < height):

            if(j > leftBlanks - 1):

                print("#", end="")

            else:

                print(" ", end="")

        if(j == height):

            print("  ", end="")

        if(j > height):

            if(j <= rightBlanks):

                print("#", end="")

    leftBlanks = leftBlanks - 1

    if(j > height):

        rightBlanks = rightBlanks + 1
    print("")

File: workspace/pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 0 || height > 23);

    int leftBlanks = height - 1;
    int rightBlanks = (height * 2) - (height - 1);
    int j;

    for (int i = 0; i < height; i++)
    {
        for (j = 0; j < (height * 2) + 2; j++)
        {
            if (j < height)
            {
                if (j > leftBlanks - 1)
                {
                    printf("#");
                }
                else
                {
                    printf(" ");
                }
            }

            if (j == height)
            {
                printf("  ");
            }

            if (j > height)
            {
                if (j <= rightBlanks)
                {
                    printf("#");
                }
            }
        }
        leftBlanks--;
        if (j > height)
        {
            rightBlanks++;
        }
        printf("\n");
    }
}